src/Components/Admins/index.js

```diff
@@ -1,9 +1,120 @@
+import { useEffect, useState } from 'react';
+import Table from './Table/index.jsx';
 import styles from './admins.module.css';
+import Form from './Form';
 
 function Admins() {
+  const [adminsData, setAdminsData] = useState([]);
+  const [showForm, setShowForm] = useState(false);
+  const [selectedAdmin, setSelectedAdmin] = useState(null);
+
+  const getAdmins = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/admins`);
+      const jsonData = await response.json();
+      const adminData = jsonData.data;
+      setAdminsData(adminData);
+    } catch (error) {
+      alert('Error getting Admins');
+    }
+  };
+
+  useEffect(() => {
+    getAdmins();
+  }, []);
+
+  const addAdmin = async ({ firstName, lastName, dni, phone, email, city, password }) => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/admins`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({ firstName, lastName, dni, phone, email, city, password })
+      });
+      const responseData = await response.json();
+      if (!responseData.error) {
+        const newAdmin = responseData.data;
+        setAdminsData([...adminsData, newAdmin]);
+        setShowForm(false);
+        setSelectedAdmin(null);
+        alert('Admin created correctly!');
+      } else {
+        throw new Error(responseData.message);
+      }
+    } catch (error) {
+      setShowForm(true);
+      console.log(error);
+      alert('Error creating admin: ' + error);
+    }
+  };
+
+  const editAdmin = async (updatedAdmin, adminId) => {
```

Comment:
This function executes when the edit button on the table is pressed, the fetch of PUT should be done when pressing the submit button of the form.

Reply:
The function has been moved the form's submit button.